Apps/Patient/src/Models/PatientModel.cs

```diff
@@ -0,0 +1,89 @@
+﻿//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+
+namespace HealthGateway.Patient.Models
+{
+    using System;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// The patient data model.
+    /// </summary>
+    public class PatientModel
```

Comment:
What do you think about keeping the Common PatientModel for the patient service as well and create a new patient model for the user interface on webclient?

Reply:
WebClient doesn't return patient data to the UI it only uses it to interface to other systems.  Patient is the one that is being directly called by VUE and was giving back the additional data not required.